src/Autofac.Extras.MvvmCross/AutofacMvxIocProvider.cs

```diff
@@ -387,7 +403,8 @@ public virtual void RegisterSingleton(Type tInterface, object theObject)
 
             // You can't inject properties on a pre-constructed instance.
             cb.RegisterInstance(theObject).As(tInterface).AsSelf().SingleInstance();
-            cb.Update(this.Container);
+
+            cb.Update(Container);
```

Comment:
Is there is a way to configure all the registrations up front using a container builder and then build the container in a single step? I don't see anything indicating such support in the `IMvxIoCProvider` interface.